### PR TITLE
Reopened: Force Username & Email to Lowercase + Remove Sensitive Data

### PR DESCRIPTION
**Pull request by almegdad**
* add directive to force username & email to lowercase
* remove sensitive data in password reset
* 2 spaces indentation  in reset & forgot password views

### CODE REVIEW

File: modules/users/client/directives/users.client.directive.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+  return {
+    require: 'ngModel',
+    link: function (scope, element, attrs, modelCtrl) {
+      modelCtrl.$parsers.push(function (input) {
+        return input ? input.toLowerCase() : '';
+      });
+      element.css('text-transform', 'lowercase');
+    }
+  };
+});
```
Comment (codydaig) [inline modules/users/client/directives/users.client.directive.js]: New Line

**Comment by codydaig** [comment]
@almegdad Your missing a new line at the end of the users.client.directive.js file. Other than that, it looks good to me!

File: modules/users/client/directives/users.client.directive.js (hunk 1 above)
Comment (almegdad) [inline modules/users/client/directives/users.client.directive.js]: updated: new line added

**Comment by codydaig** [comment]
LGTM

**Pull request merged**